### ISSUE TITLE
 Extend peer dependencies to Angular 8

### ISSUE THREAD

**Issue by alvaro-octal** [webcat12345/ngx-ui-switch#349]
**Issue description**

running `npm install` throws the following warning:

```
npm WARN ngx-ui-switch@8.0.1 requires a peer of @angular/core@>=5.0.0 <8.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-ui-switch@8.0.1 requires a peer of @angular/common@>=5.0.0 <8.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-ui-switch@8.0.1 requires a peer of @angular/forms@>=5.0.0 <8.0.0 but none is installed. You must install peer dependencies yourself.
```
**Steps to reproduce and a minimal demo of the problem**

`npm install` with Angular 8

**Current behavior**

`npm install` shows a couple of warning due to peer dependencies.

**Expected/desired behavior**

No warnings should be shown

**Comment by FrancescoBorzi** [comment]
Confirmed

**Comment by cmckni3** [comment]
yeah ng 8 has been out for just over a day. I'll see if I can get to it tomorrow.

PRs are welcome.

**Comment by cmckni3** [comment]
Until then ngx-ui-switch should still function properly outside of the peer dependency warnings.

**Comment by FrancescoBorzi** [comment]
the issue is about the warning and I would keep it open until it is actually solved

**Comment by alvaro-octal** [comment]
It's solved, checkout:

```
npm install ngx-ui-switch@8.1.0 --save
```

**Comment by cmckni3** [comment]
> It's solved, checkout:
> 
> ```
> npm install ngx-ui-switch@8.1.0 --save
> ```

yep, forgot to mention I released [8.1.0](https://github.com/webcat12345/ngx-ui-switch/releases/tag/8.1.0)
